attributes/default.rb

```diff
@@ -1,5 +1,11 @@
 
 # Override cookbook attributes to install Java 7
-override['java']['openjdk_packages'] = [
-  "openjdk-7-jdk", "openjdk-7-jre-headless"
-]
+#override['java']['openjdk_packages'] = [
+#  "openjdk-7-jdk", "openjdk-7-jre-headless"
+#]
```

Comment:
I removed this as it seems to not be needed. I had no problem getting the spigot server running without this. Thoughts?